init.g

```diff
@@ -46,7 +46,9 @@ ReadPackage("GAPDoc", "lib/Examples.gd");
 
 
 # try to find terminal encoding
-GAPInfo.tmpfunc := function()
+CallFuncList(function()
```

Comment:
It is debatable what the nicer syntax is. (In new enough GAP on can use `function() ... end();` which is certainly nicer; but I don't want to create a dependency on such new versions of GAP at this stage.)

But independent of this question this change does not make any sense. How can `GAPInfo` be read-only? Its purpose is to hold information about the current GAP session and this can change during the session (e.g., by loading a package). And in particular this function exists to set the entry `GAPInfo.TermEncoding` which is impossible anyway if `GAPInfo` is read-only. 

I don't see how  this change can be useful for HPCGAP.